crates/swc_ecma_parser/src/lexer/token.rs

```diff
@@ -1469,6 +1469,17 @@ impl Token {
             _ => false,
         }
     }
+
+    pub(crate) fn should_rescan_into_gt_in_jsx(self) -> bool {
```

Comment:
The current implementation works well, but for reference, TypeScript employs an alternative approach:
- Only eat the greater symbol in `/`/`>=`/`>>`/`>>=`.
- And then rescan again when parsing binary expression.

The approach means we don't need to rescan much time during parse jsx expression